app/views/application/references/alternative-review.njk

```diff
@@ -0,0 +1,81 @@
+{% extends "_content-wide.njk" %}
+
+{% set applicationPath = "/application/" + applicationId %}
+{% set referrer = applicationPath + "/references/review" %}
+
+{% block pageNavigation %}
+  {{ govukBackLink({
+    href: applicationPath + "/references/alternative",
+    text: "Back"
+  }) }}
+{% endblock %}
+
+{% block primary %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Check your answers before&nbsp;confirming</h1>
```

Comment:
‘Check the referee's details before submitting’

Reply:
Going for "Check your referee’s details before confirming"